tests/testthat/assets/home-index-template-md/inst/pkgdown/templates/content-home.html

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="contents col-md-9">
+    {{{index}}}
```

Comment:
You may need to deindent the `{{{}}}` content here.

Reply:
Yes, the indentation was causing the issue with the homepage and also appeared in this test. Rendered code blocks in the Rmarkdown were converted to html with `pre` tags, and the indentation was then added. I've deindented this in the template, and the two custom templates in the tests.